### DISCUSSION TITLE
Dashboard and Reports data mismatch issue in Production

### DISCUSSION THREAD

**Discussion by pallavii2809** [egovernments/Digit-Core#132] [Dashboard]
Regarding the Dashboard currently data is mismatched between dashboard and reports on Production server
How can we get the data matched between the Dashboard and the report without deleting anything from the database?

**Comment by pradeepkumarcm-egov** [comment]
The dashboards are working using the elastic search indexes. Each module push data to eleastic search on create/update of entry.Re-index the data using plain search service to get all the data to the respective indexer. 

We have 2 steps for this. The first is to run the connector from the playground, which is followed by the legacyindexer service call from the indexer service, which internally calls the respective plain search service to get the data and send it to the respective indexer.

Refer mCollect reindexing steps [here](https://urban.digit.org/platform/configure-digit/configuring-digit-services/mcs-service-configuration/re-indexing-the-mcollect-service). 

We mentioned the reindexing steps for other modules in urban.digit.org.

**Reply by pallavii2809** [reply]
Hii @pradeepkumarcm-egov we are facing this issue while we are trying to hit the legacyindex API for pgr-services

![image](https://github.com/egovernments/Digit-Core/assets/109335293/621f90b2-4d30-4c38-955d-aec62e20d0cc)

**Comment by pradeepkumarcm-egov** [comment]
Please check the PGR index files are present in the path. Refer https://github.com/egovernments/configs/blob/273165b48dd911f4c7a487b02d50f88356b01ab7/egov-indexer/pgr-services.yml#L107

Check the index paths are marked in egov-indexer path. Refer: https://github.com/egovernments/DIGIT-DevOps/blob/37235f6e1a9a1915995aa9493a00bd350fac735b/deploy-as-code/helm/environments/dev.yaml#L246
